relay-general/src/store/normalize.rs

```diff
@@ -221,12 +236,42 @@ fn normalize_breakdowns(event: &mut Event, breakdowns_config: Option<&Breakdowns
     }
 }
 
-/// Ensure measurements interface is only present for transaction events
-fn normalize_measurements(event: &mut Event) {
+/// Enforce the limit on custom (user defined) measurements.
+///
+/// Note that [`Measurements`] is a BTreeMap, which means its keys are sorted.
+/// This ensures that for two events with the same measurement keys, the same set of custom
+/// measurements is retained.
+///
```

Comment:
Good comment 👍 (just a nit on the extra line)
```suggestion
/// Note that [`Measurements`] is a BTreeMap, which means its keys are sorted.
/// This ensures that for two events with the same measurement keys, the same set of custom
/// measurements is retained.
```